### PR TITLE
TTL on the message directories

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n i have tried to add TTL as attribute in ETCDOBJ but other child object classes gives exception like

**Comment by GowthamShanmugam** [comment]
@shtripat @nthomas-redhat please review

File: tendrl/commons/etcdobj/__init__.py [hunk 1]
```diff
@@ -70,15 +70,18 @@ def _verify_client(self, client):
...
 
         :param obj: An instance that subclasses EtcdObj
         :type obj: EtcdObj
         :returns: The same instance
         :rtype: EtcdObj
         """
-        for item in obj.render():
+        render = obj.render()
+        if ttl != None:
+            NS.etcd_orm.client.write(obj.__name__, '', dir=True, ttl=ttl)
```
Comment (r0h4n) [inline tendrl/commons/etcdobj/__init__.py]: if ttl:
   self.client.write(obj.__name__, '', dir=True, ttl=ttl)

Comment (GowthamShanmugam) [inline tendrl/commons/etcdobj/__init__.py]: @r0h4n i will change it

**Pull request merged**